website/docs/add-secure-apps/providers/saml/IDP-initiated-single-logout.md

```diff
@@ -0,0 +1,28 @@
+---
+title: SAML IDP Initiated Single Logout
```

Comment:
```suggestion
title: SAML IDP-initiated single logout
```

Reply:
This PR has made me realize that we are all over the place with the capitalization on titles, in the Providers section. Our Style Guide calls for Sentence case capitalization in titles. I'll do a separate PR for that, across all providers.